CMakeLists.txt

```diff
@@ -9,17 +9,13 @@
 
 cmake_minimum_required( VERSION 3.12 )
 
-set(iodaconv_major 0)
-set(iodaconv_minor 0)
-set(iodaconv_revision 1)
-set(iodaconv_version ${iodaconv_major}.${iodaconv_minor}.${iodaconv_revision})
-project( iodaconv VERSION ${iodaconv_version} LANGUAGES Fortran )
+project( iodaconv VERSION 0.0.1 LANGUAGES C CXX Fortran )
```

Comment:
I don't think we need `C` language compiler, just `CXX`, right?

Reply:
Ok.